packages/kotti-ui/source/kotti-field-file-upload/validators.ts

```diff
@@ -0,0 +1,70 @@
+import { KottiFieldFileUpload } from './types'
+
+const isValidFileSize = (
+	fileSize: number,
+	maxFileSize: number | null,
+): boolean => maxFileSize === null || fileSize < maxFileSize
+
+const isValidFileType = (
+	allowedFileTypes: string | null,
```

Comment:
weird choice of types - why support the extensions argument as a concatenated string vs an array?

Reply:
i mean not even a regex, right? it ends being a lower-case string that we use "include" on

which is very risky

I mean - theoratically - we could end up getting the end of a string and the beginning of another as a format

--

Feels like legacy implementation coming from yoda because i remember seeing this before but I would totally advise against this :/

Reply:
Not really a preference, either way will have to check the strings as we could have almost any input
I refactored it to an array pull latest changes and let me know what you think